main/index.ts

```diff
@@ -4,6 +4,8 @@ import log from 'electron-log'
 import { numberToHex } from 'web3-utils'
 import url from 'url'
 
+process.env.BUNDLE_LOCATION = process.env.BUNDLE_LOCATION || path.resolve(__dirname, './../..', 'bundle')
```

Comment:
can you add a small comment about why this needs to be here so it doesn't get moved?

Reply:
good idea